README.md

```
@@ -120,4 +133,56 @@
+- `main`
+  - 프로젝트 최종 merge
+  - 기본 프로젝트 세팅, 배포 가능한 브랜치, 항상 배포 가능한 상태를 유지
+- `develop`
+  - 데모데이 전까지 완성한 기능들을 계속해서 merge
+  - 배포 가능한 브랜치, 항상 배포 가능한 상태를 유지
+- `{type}/{description}`: 개발 브랜치
+  - 예: `feat/login`, `fix/login-token`
+
+&nbsp;
+## 👩‍💻👨‍💻 Organization Repository
+https://github.com/Farm-On/BE
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)